package.json

```diff
@@ -21,7 +25,9 @@
     "react": "^18",
     "react-dom": "^18",
     "socket.io": "^4.7.5",
-    "socket.io-client": "^4.7.5"
+    "socket.io-client": "^4.7.5",
+    "tailwind-merge": "^2.5.2",
+    "tailwindcss-animate": "^1.0.7"
```

Comment:
I don't know about `tailwindcss-merge` (it might be), but `tailwindcss-animate` is definitely a dev dependency.
see: https://www.npmjs.com/package/tailwindcss-animate#installation